src/screens/AuthScreen.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import Login from '../components/Auth/Login';
+import Register from '../components/Auth/Register';
+import { Button, Card, CardActions } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 500,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    margin: '20% auto'
+  }
+});
+
+export default function AuthScreen() {
+  const [state, setState] = useState('home');
```

Comment:
I think eventually we can try to do the routing at the top level component, instead of AuthScreen. The root component could decide on which component to render, based on the currently logged in user, so that AuthScreen doesn't need to keep state about home/login/register itself.